owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.java

```diff
@@ -133,6 +137,39 @@ protected void setupDrawer() {
                 mNavigationView.setItemIconTintList(null);
             }
 
+            //Notch support
+            mNavigationView.addOnAttachStateChangeListener(new View.OnAttachStateChangeListener() {
+                @Override
+                public void onViewAttachedToWindow(View v) {
+
+                    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+
+                        DisplayCutout displayCutout = v.getRootWindowInsets().getDisplayCutout();
+
+                        if (displayCutout != null) {
+                            RelativeLayout rl_drawer_active_user =
+                                    (RelativeLayout) findNavigationViewChildById(R.id.drawer_active_user);
+
+                            int orientation = getResources().getConfiguration().orientation;
+                            if (orientation == Configuration.ORIENTATION_PORTRAIT) {
+                                int displayCutoutDP = (displayCutout.getSafeInsetTop()) /
+                                                (getResources().getDisplayMetrics().densityDpi /
+                                                        DisplayMetrics.DENSITY_DEFAULT);
+                                rl_drawer_active_user.getLayoutParams().height =
```

Comment:
We normally use camelCase for variable names and _ for layout ids for example, as you can see in  `(RelativeLayout) findNavigationViewChildById(R.id.drawer_active_user);`